CHANGES.rst

```diff
@@ -1,6 +1,25 @@
 Changes
 =======
 
+Version 0.5.0 (2019-04-16)
+--------------------------
+
+- Introduces new workflow statuses: ``deleted``, ``stopped``, ``queued``.
```

Comment:
It would be good to order the news by topics, e.g. start with all workflow-related changes, continue with session-related changes, and end up with job-related changes.  (top-to-bottom kind of direction)